common/types/pb/pb.go

```diff
@@ -41,7 +41,7 @@ func DescribeFile(message proto.Message) (*FileDescription, error) {
 	if fd, found := revFileDescriptorMap[proto.MessageName(message)]; found {
 		return fd, nil
 	}
-	fileDesc, _ := descriptor.ForMessage(message.(descriptor.Message))
+	fileDesc, _ := descpb.ForMessage(message.(descpb.Message))
```

Comment:
There are issues in the Travis build related to an unused import for 'descriptor', so you'll have to add the alias 'descpb' to that import statement. The default go import strategy is to use the simple name of the last directory in the import which explains the error in the logs.